pkg/kubectl/stop_test.go

```diff
@@ -18,46 +18,253 @@ package kubectl
 
 import (
 	"fmt"
+	"reflect"
 	"testing"
 	"time"
 
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/client"
 	"k8s.io/kubernetes/pkg/client/testclient"
+	"k8s.io/kubernetes/pkg/runtime"
 )
 
-func TestReplicationControllerStop(t *testing.T) {
+func TestReplicationControllerStopTable(t *testing.T) {
```

Comment:
I don't think the name of this test needs to change. We have lots of table tests and don't usually reflect as such in the name.